.github/workflows/workflow.yaml

```diff
@@ -215,7 +212,7 @@ jobs:
         run: |
           mkdir -p $GITHUB_WORKSPACE/bin
           echo "$GITHUB_WORKSPACE/bin" >> $GITHUB_PATH
-          make e2e-bootstrap KUBERNETES_VERSION=${{ matrix.KUBERNETES_VERSION }}
```

Comment:
We are able to remove this b/c it's taken from the environment variables?

Reply:
that's right, no reason to keep this unless we are using as matrix